src/PlaybackManager.vala

```diff
@@ -60,6 +65,35 @@ public class Music.PlaybackManager : Object {
         notify["current-audio"].connect (on_audio_changed);
 
         settings = new Settings ("io.elementary.music");
+
+        var clear_action = new SimpleAction (Application.ACTION_CLEAR_QUEUE, null);
+        clear_action.activate.connect (clear_queue);
+
+        next_action = new SimpleAction (Application.ACTION_NEXT, null);
+        next_action.activate.connect (() => next ());
+
+        play_pause_action = new SimpleAction.stateful (Application.ACTION_PLAY_PAUSE, null, new Variant.boolean (false));
+        play_pause_action.change_state.connect (play_pause);
+
+        previous_action = new SimpleAction (Application.ACTION_PREVIOUS, null);
+        previous_action.activate.connect (previous);
+
+        shuffle_action = new SimpleAction (Application.ACTION_SHUFFLE, null);
+        shuffle_action.activate.connect (shuffle);
+
+        next_action.set_enabled (false);
+        play_pause_action.set_enabled (false);
+        previous_action.set_enabled (false);
+        shuffle_action.set_enabled (false);
+
+        var action_group = GLib.Application.get_default ();
+        action_group.add_action (clear_action);
+        action_group.add_action (next_action);
+        action_group.add_action (play_pause_action);
+        action_group.add_action (previous_action);
+        action_group.add_action (shuffle_action);
```

Comment:
I feel this is strange with the variable name because the interface that has `add_action()` is not `GLib.ActionGroup` but `GLib.ActionMap`.

But anyways I think we don't need to think about which interface has it and simply name as `app`. Also explicit `unowned`

```suggestion
        unowned var app = GLib.Application.get_default ();
        app.add_action (clear_action);
        app.add_action (next_action);
        app.add_action (play_pause_action);
        app.add_action (previous_action);
        app.add_action (shuffle_action);
```